splinepy/multipatch.py

```diff
@@ -270,10 +332,10 @@ def determine_interfaces(self):
 
         return self.interfaces
 
-    def add_boundary_with_function(
+    def boundary_from_function(
         self,
         function,
-        only_unassigned=False,
+        from_boundaries=None,
```

Comment:
```suggestion
        mask=None,
```
boundaries from function from boundaries is a bit unclear